SecondSight/main.py

```diff
@@ -31,6 +30,7 @@ def mainLoop():
     # TODO: Every module should write to network tables, not from here
     networktables.NetworkTables.initialize(server=config.get_value('nt_dest'))
     april_table = networktables.NetworkTables.getTable('SecondSight').getSubTable('Apriltags')
+    conecube_table = networktables.NetworkTables.getTable('SecondSight').getSubTable('GamePieces')
```

Comment:
I'm not keen on this being in main. Don't worry about it now, but we should think about how to do it better

Reply:
It needs to be created somewhere, and fetching the table every time would be time-consuming. There is no better place to put it.